community/kernel/src/main/java/org/neo4j/kernel/DatabaseAvailability.java

```diff
@@ -34,11 +37,19 @@
 {
     private final AvailabilityGuard availabilityGuard;
     private final TransactionCounters transactionMonitor;
+    private final long awaitActiveTransactionDeadlineMillis;
 
     public DatabaseAvailability( AvailabilityGuard availabilityGuard, TransactionCounters transactionMonitor )
     {
+        this( availabilityGuard, transactionMonitor, TimeUnit.SECONDS.toMillis( 10 ) );
```

Comment:
Have we experimented with this timeout? 10 ms sounds short, but I don't have the whole picture of what it means.